packages/common-server/src/analytics.ts

```diff
@@ -256,6 +256,7 @@ export class SegmentClient {
       this._anonymousId = genUUID();
       fs.writeFileSync(uuidPath, this._anonymousId);
     }
+    this.logger.info({ msg: "anonymous id", anonymousId: this._anonymousId });
```

Comment:
super nit: maybe "anonymousId" as one-word will be easier to find via Ctrl+F since we're more accustomed to having it camel case (in the code anyway)

Reply:
You can still `Ctrl+F` it since it's also the property name (`anonymousId: this._anonymousId`)